src/index.css

```diff
@@ -0,0 +1,29 @@
+
+.offscreen {
```

Comment:
Instead of this, can you just set `display: none` ?

Reply:
I will use screen reader for accessibility later so maybe it's better to use .offscreen?